src/tests/v110/test_list_collections.py

```diff
@@ -0,0 +1,23 @@
+import fireo
+from fireo.fields import IDField
+from fireo.models import Model
+
+
+class Doc(Model):
+    doc_id = IDField(db_name='id')
```

Comment:
First of all there is no attribute `db_name` it's `column_name` and second thing `IDField` is not support `column_name` also because `IDField` is not saved explicitly with document **it is actually a document id not document field** 
Please read the documentation to see how FireO work [Documentation](https://octabyte.io/FireO/fields/id-field/)

Reply:
@AxeemHaider I synced to run Travis.  Didn't realize it would update the PR.  You can ignore these at is work in progress.